packages/plugin-core/assets/dendron-ws/tutorial/treatments/quickstart-v1/tutorial.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
I did a first pass copy of the quickstart. This is synced from https://github.com/dendronhq/dendron-site/pull/527

If you have any feedback, please leave it in that PR.